client/components/jetpack/alert-banner/index.tsx

```diff
@@ -0,0 +1,13 @@
+import React, { ReactNode } from 'react';
+import './style.scss';
+
+type AlertBannerProps = {
+	type: 'warning' | 'error' | 'success' | 'info';
+	children: ReactNode;
+};
+
+const AlertBanner: React.FC< AlertBannerProps > = ( { type, children } ) => {
+	return <div className={ `alert-banner alert-banner--${ type }` }>{ children }</div>;
+};
+
+export default AlertBanner;
```

Comment:
I have a suggestion here

We usually keep one extra line before import './style.scss';

```suggestion
import classNames from 'classnames';
import React, { ReactNode } from 'react';

import './style.scss';

type AlertBannerProps = {
	type: 'warning' | 'error' | 'success' | 'info';
	children: ReactNode;
};

const AlertBanner: React.FC< AlertBannerProps > = ( { type, children } ) => {
	const alertBannerClasses = classNames( 'alert-banner', {
		'alert-banner__warning': type === 'warning',
		'alert-banner__error': type === 'error',
		'alert-banner__success': type === 'success',
		'alert-banner__info': type === 'info',
	} );

	return <div className={ alertBannerClasses }>{ children }</div>;
};

export default AlertBanner;

```